src/modules/qt/transition_qtblend.cpp

```diff
@@ -25,6 +25,16 @@
 #include <QPainter>
 #include <QTransform>
 
+static int is_opaque( uint8_t *image, int width, int height )
```

Comment:
Since this appears in `transition_frei0r.c` I think it should be refactored into the mlt_image class simply for convenience but not to require a `mlt_image` struct as that would be inefficient. Brian has a goal to include mlt_image pointers on a new get_image callback someday. So, we need to anticipate that someday there will be a `mlt_image_is_opaque()` that takes a `mlt_image` pointer. Thus, lets call this one `mlt_image_rgba_opaque()`.

Also, Brian and I recently learned that compilers can optimize a for loop with buffer array indexing better than a while loop with pointer arithmetic. So, we ought to refactor for that as well:
```
int mlt_image_rgba_opaque(uint8_t *image, int width, int height)
{
	for (int i = 0; i < width * height; ++i) {
		if (image[4 * i + 3] != 0xff) return 0;
	}
	return 1;
}
```
And do not forget to add it to `mlt.vers`.